website/sidebars.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  docs: {
+    Introduction: ["intro", "installation", "limitations"],
+    Commands: [
+      {
+        Scan: ["cmd/scan/iac-source", "cmd/scan/output", "cmd/scan/filtering"],
+        Flags: ["cmd/flags/error-reporting", "cmd/flags/version-check"],
```

Comment:
Theses are not really commands, maybe we should move theses one outside of this section

Reply:
What do you have in mind ?

We could create a `Usage` section with two subsections `Commands` and `Flags`. WDYT

Reply:
> What do you have in mind ?

Nothing atm, I was looking around in other OSS project doc how they managed their tree.

> We could create a Usage section with two subsections Commands and Flags. WDYT

Sounds better, I don't know if a `Flags` section is clear enough for users.